From ba42dd72ce1a290a08dc88efe60b65b5d3320b66 Mon Sep 17 00:00:00 2001
Message-Id: <ba42dd72ce1a290a08dc88efe60b65b5d3320b66.1434058173.git.chang-joon.lee@intel.com>
In-Reply-To: <cb13d6abcebb263859514735a399f7f8b895c27a.1434058173.git.chang-joon.lee@intel.com>
References: <cb13d6abcebb263859514735a399f7f8b895c27a.1434058173.git.chang-joon.lee@intel.com>
From: Akashdeep Sharma <akashdeep.sharma@intel.com>
Date: Mon, 1 Jun 2015 19:23:30 +0530
Subject: [PATCH 3/6] SQUASHME! [VPG]: drm: Use right level of log message in
 set property ioctl

Change debug level of log messages while obtaining single Crtc
lock in set property ioctl. Use DRM_DEBUG_KMS instead of DRM_ERROR
while dumping these logs. Otherwise, these log messages will be
reflected as Errors in dmesg/ serial logs giving wrong impression
to others. Also this causes unnecessary delay of few milliseconds,
delays flips and might appear as stutter during video playback.

This patch should be squashed into the following patch:

commit e79bcf7a2a9342f471a49a2a6403f65ff0227a9e
change-Id: Ife6cba9fc6338966d6b8924544b52437389946c3
Author: Akashdeep Sharma <akashdeep.sharma@intel.com>
Date:   Wed May 20 00:27:40 2015 +0530
    REVERTME [VPG]: drm: Take single CRTC lock

Change-Id: I73e94c6541f4621d86371cadd580dcdfc33c50d3
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-31068
Signed-off-by: Akashdeep Sharma <akashdeep.sharma@intel.com>
---
 drivers/gpu/drm/drm_crtc.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/drm_crtc.c b/drivers/gpu/drm/drm_crtc.c
index 68d521f..858936d 100644
--- a/drivers/gpu/drm/drm_crtc.c
+++ b/drivers/gpu/drm/drm_crtc.c
@@ -4197,15 +4197,15 @@ int drm_mode_obj_set_property_ioctl(struct drm_device *dev, void *data,
 	case DRM_MODE_OBJECT_CONNECTOR:
 		crtc = drm_crtc_from_connector(obj_to_connector(arg_obj));
 		if (crtc) {
-			DRM_ERROR("CRTC from connector,CRTC={id=%d props=%d}\n",
-				crtc->base.id, crtc->base.properties ?
+			DRM_DEBUG_KMS("CRTC from connector,CRTC={id=%d props=
+				%d}\n", crtc->base.id, crtc->base.properties ?
 					crtc->base.properties->count : 0);
 			mutex_lock(&config->mutex);
 			drm_modeset_lock(&crtc->mutex, NULL);
 			ret = drm_mode_connector_set_obj_prop(arg_obj, property,
 							arg->value);
 		} else {
-			DRM_ERROR("No crtc from connector, lock all\n");
+			DRM_DEBUG_KMS("No crtc from connector, lock all\n");
 			drm_modeset_lock_all(dev);
 			ret = drm_mode_connector_set_obj_prop(arg_obj, property,
 							arg->value);
-- 
1.9.1

